vins_estimator/src/factor/integration_base.h

```diff
@@ -3,6 +3,7 @@
 #include "../utility/utility.h"
 #include "../parameters.h"
 
+#include "spdlog/spdlog.h"
```

Comment:
One thing we could note is, in future when we clean up the codebase to a point where all implementation details are only found in .cpp files, then `#include "spdlog/spdlog.h"` will only be found in .cpp files and we can do target_link_libraries(libraryXXX ... PRIVATE spdlog) to hide the spdlog include header files from the user of our libraries. This should be cleaner

Reply:
Agreed!